sdk/attestation/attestation/README.md

```diff
@@ -180,13 +180,27 @@ InitTime data refers to data which is used to configure the SGX enclave being at
 Creates an instance of the Attestation Client at uri `endpoint`.
 
 ```ts
-<FILL THIS IN>
+  const client = new AttestationClient(new DefaultAzureCredential(), endpoint);
```

Comment:
There is still a few `<FILL THIS IN>` left

Reply:
Known - I don't have all the samples completed.

Reply:
Do you plan to fill them in this PR?